addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19464,7 +19464,31 @@ msgctxt "#38042"
 msgid "[Missing]"
 msgstr ""
 
-#empty strings from id 38043 to 38099
+#. Title album artists node
+#: system/library/music/musicroles/albumartists.xml
+msgctxt "#38043"
+msgid "Album Artists"
+msgstr ""
+
+#. Title all artists node
+#: system/library/music/musicroles/allartists.xml
+msgctxt "#38044"
+msgid "Song & Album Artists"
+msgstr ""
+
+#. Title all contributors node (any role)
+#: system/library/music/musicroles/allcontributors.xml
+msgctxt "#38045"
+msgid "All Contributors"
```

Comment:
Just my natural habit to capitalise titles, no idea of the Kodi design policy.  Was that a question for others to confirm or rhetorical (meaning change to lower case).?